cmd/plugins/memtierd/Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG GO_VERSION=1.21
```

Comment:
nit: other dockerfiles have 1.20 so I suggest to be aligned with that
```suggestion
ARG GO_VERSION=1.20
```

Reply:
Fixed.